app/services/card_comparator/result.rb

```diff
@@ -0,0 +1,14 @@
+class CardComparator::Result < Struct.new(:diff)
+
+  def success?
+  	diff == 0
+  end
+
+  def type_error?
+  	diff.between?(0, 2) 
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Tab detected.<br>Trailing whitespace detected.